File: praktikum_6/PRAK605-2410817210004-Rahmad.py
ordo=int(input(""))

matriks_A=[]
matriks_B=[]
#meminta user menginput isi dari matriks A

#for i in range(ordo):
#    baris_matriks = []
#    for j in range(ordo):
#        elemen = int(input(" "))
#        baris_matriks.append(elemen)
#    matriks_A.append(baris_matriks)
##meminta user menginput isi dari matriks B
#for i in range(ordo):
#    baris_matriks = []
#    for j in range(ordo):
#        elemen = int(input(" "))
#        baris_matriks.append(elemen)
#    matriks_B.append(baris_matriks)
for i in range(ordo):
    baris_matriks = list(map(int, input().split()))  # Membaca input satu baris dan mengubahnya menjadi list of integers
    matriks_A.append(baris_matriks)
for i in range(ordo):
    baris_matriks = list(map(int, input().split()))  # Membaca input satu baris dan mengubahnya menjadi list of integers
    matriks_B.append(baris_matriks)


#operasi perkalian

hasil=[]
for i in range(ordo):
    baris_hasil =[]
    for j in range(ordo):
        perkalian=0
        for k in range(ordo):
            perkalian+=matriks_A[i][k]*matriks_B[k][j]
        baris_hasil.append(perkalian)
    hasil.append(baris_hasil)

#output
for row in hasil :
    print(row)

File: praktikum_6/PRAK605-2410817210004-Rahmad.c
#include <stdio.h>
int main(){
    int ordo ;
    //meminta user memasukan inputan ordo
    scanf("%d",&ordo);

    int matriks_A[ordo][ordo];
    int matriks_B[ordo][ordo];
    //ini meminta user menginput isi dari matriks A
    for(int i= 0;i<ordo;i++){
        for(int j= 0;j<ordo;j++){
        scanf("%d",&matriks_A[i][j]);
        }
    }
    //ini meminta user menginput isi dari matriks B
    for(int i= 0;i<ordo;i++){
        for(int j= 0;j<ordo;j++){
        scanf("%d",&matriks_B[i][j]);
        }
    }
    //perkalian matriks
    int hasil[ordo][ordo];
 
    for(int i= 0;i<ordo;i++){
        for(int j= 0;j<ordo;j++){
            hasil[i][j]=0;
            for(int k=0;k<ordo;k++){
        hasil[i][j]+=matriks_A[i][k]*matriks_B[k][j];
        }
        }
    } 
    for(int i= 0;i<ordo;i++){
        for(int j= 0;j<ordo;j++){
           printf("%d ",hasil[i][j]);
        }printf("\n");
    } 
}   